build_tools/travis/flake8_diff.sh

```diff
@@ -125,10 +125,13 @@ MODIFIED_FILES="$(git diff --name-only $COMMIT_RANGE | grep -v 'sklearn/external
 
 check_files() {
     files="$1"
-    options="$2"
-    # Conservative approach: diff without context (--unified=0) so that code
-    # that was not changed does not create failures
-    git diff --unified=0 $COMMIT_RANGE -- $files | flake8 --diff --show-source $options
+    shift
+    options="$*"
```

Comment:
While I was at it I made the function more robust to be able to take multiple options. cc @jnothman.